guides/assets/stylesrc/style.scss

```diff
@@ -48,7 +48,7 @@ $gray-500: #A9A9A8;
 $gray-600: #666666;
 $gray-700: #3A3939;
 $gray-800: #343434;
```

Comment:
On mobile, there are sections that use gray-800 as the background, so we might want to update it too.

---

Current 3 shades of gray:
<img width="387" alt="image" src="https://github.com/rails/rails/assets/20938712/0e8f64f8-91ec-4d1b-a87a-e196870ee108">